### PR TITLE
Archive notice links to newer version of the same page

### PR DESCRIPTION
**Pull request by timstclair**
Fixes https://github.com/kubernetes/kubernetes/issues/17129

### CODE REVIEW

**Comment by timstclair** [comment]
cc/ @RichieEscarez

File: _includes/archivedocnotice.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <div>
 	<br/>
-	<p style="text-align:center; border-style:ridge; border-width:2px; border-color:red">This is an archived version of the documentation. <a href="http://kubernetes.io/docs">View the latest version here.</a></p>
+	<p style="text-align:center; border-style:ridge; border-width:2px; border-color:red">This is an archived version of the documentation. <a id="archivednotice" href="{{site.url}}{{ page.url | replace: '/v1.0/', '/v1.1/' }}">View the latest version here.</a></p>
```
Comment (janetkuo) [inline _includes/archivedocnotice.html]: What if we more than 2 versions in the future?

**Comment by janetkuo** [comment]
cc @caesarxuchao

**Comment by caesarxuchao** [comment]
@timstclair @janetkuo Thank you for taking care of this.

File: _includes/archivedocnotice.html (hunk 1 above)
Comment (timstclair) [inline _includes/archivedocnotice.html]: Done.

**Comment by janetkuo** [comment]
LGTM now. @caesarxuchao can you take a look too?

**Comment by caesarxuchao** [comment]
LGTM. Thank you.

**Comment by caesarxuchao** [comment]
I'll manually merge and watch the website.

**Comment by caesarxuchao** [comment]
@timstclair This still has a problem. Sometimes the archive notice link points to a page browsed earlier. Probably it's because the page.url doesn't get updated until the user manually refreshes the page, because we only update the doc part of the page when a user clicks on a link.

**Comment by caesarxuchao** [comment]
I can't find a quick fix. @timstclair @janetkuo do you have time to fix this?

**Comment by timstclair** [comment]
I think this is ok for now. I'll fix it tomorrow.

**Pull request merged**